js/lib/Transaction.js

```diff
@@ -1,36 +1,12 @@
 const web3Utils = require('web3-utils');
-const ethjsUtil = require('ethereumjs-util')
+const ethjsUtil = require('ethereumjs-util');
 
-const RLP = require('rlp');
-
-class Signature {
-    constructor(v, r, s) {
```

Comment:
what if instead of defining this constructor with three params we defined it with only one param which would be an object and thus reducing the arity to just one. Also by using an object as a param we would benefit from named params since we wouldn't need to declare the params in strict order and also the reader would easily identify what the params mean

```js
constructor({v = "", r = "", s = ""} {...}
// and then when instantiating
new Signature({v: "0x0", r: "0x0", s: "0x0"})
```

Reply:
great idea